### PR TITLE
mes-10108: implement firefox for unit tests

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/delegated-rekey-upload-outcome/__tests__/delegated-rekey-upload-outcome.spec.ts
Comment (RLCorp) [inline src/app/pages/delegated-rekey-upload-outcome/__tests__/delegated-rekey-upload-outcome.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/app/__tests__/sentry-error-handler.spec.ts
Comment (RLCorp) [inline src/app/__tests__/sentry-error-handler.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/app/pages/office/components/alternate-mot-evidence-description/__tests__/alternate-evidence-description.component.spec.ts
Comment (RLCorp) [inline src/app/pages/office/components/alternate-mot-evidence-description/__tests__/alternate-evidence-description.component.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/app/providers/device-authentication/__tests__/device-authentication.spec.ts
Comment (RLCorp) [inline src/app/providers/device-authentication/__tests__/device-authentication.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/app/providers/test-persistence/__tests__/test-persistence.spec.ts
Comment (RLCorp) [inline src/app/providers/test-persistence/__tests__/test-persistence.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/components/common/adi3-debrief-card/__tests__/adi3-debrief-card.spec.ts
Comment (RLCorp) [inline src/components/common/adi3-debrief-card/__tests__/adi3-debrief-card.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/components/common/error-message/__tests__/error-message.spec.ts
Comment (RLCorp) [inline src/components/common/error-message/__tests__/error-message.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/components/common/fit-marker-banner/__tests__/fit-marker-banner.spec.ts
Comment (RLCorp) [inline src/components/common/fit-marker-banner/__tests__/fit-marker-banner.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/components/common/screen-lock-indicator/__tests__/screen-lock-indicator.spec.ts
Comment (RLCorp) [inline src/components/common/screen-lock-indicator/__tests__/screen-lock-indicator.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/store/app-info/__tests__/app-info.effects.spec.ts
Comment (RLCorp) [inline src/store/app-info/__tests__/app-info.effects.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

File: src/store/examiner-records/__tests__/examiner-records.effects.spec.ts
Comment (RLCorp) [inline src/store/examiner-records/__tests__/examiner-records.effects.spec.ts:1]: This file isn't tagged with the ticket to reintroduce tests

**Review by RLCorp — APPROVED** [review]

**Review by gittins7 — APPROVED** [review]

**Pull request merged**